internal/edgedbtypes/uuid.go

```diff
@@ -22,6 +22,24 @@ import (
 	"strings"
 )
 
+// ParseUUID parses s into a UUID or returns an error.
+func ParseUUID(s string) (UUID, error) {
+	s = strings.Replace(s, "-", "", 4)
```

Comment:
I'd add validation of string length here -- must be 32.

Reply:
changed